pages/bookmarks.jsx

```diff
@@ -73,6 +74,12 @@ class Bookmarks extends React.Component{
     });
   }
 
+  showBookmarkPrompt(show = false) {
```

Comment:
This is kind of confusing to me, especially with logic inside of the method that might actually hide the bookmark?

I usually make methods like this not have any logic and just do what they are called (eg: `showBookmarkPrompt` and `hideBookmarkPrompt`) and then put logic elsewhere.